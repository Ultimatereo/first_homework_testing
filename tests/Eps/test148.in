4
1.23e-05 1.33e-05 3.94e-05 4.66e-05 1.8e-05
2.56e-05 9e-06 3.29e-05 3.22e-05 2.5e-06
3.19e-05 2.15e-05 2.39e-05 1.3e-06 2.12e-05
2.3e-06 1e-06 8e-07 3.9e-05 2.09e-05
