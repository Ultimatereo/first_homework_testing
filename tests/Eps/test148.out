-0.278339
2.29686
-0.835445
0.510556
